test/oranif_eunit.erl

```diff
@@ -45,7 +45,7 @@ iozip(List) -> lists:zip(List, lists:seq(1, length(List))).
 
 %% tests conn_prepareStmt stmt_execute, stmt_fetch, stmt_getQueryValue,
 %% data_get, data_release and stmt_release
-stmt_execute_test({Safe, _Context, Conn}) ->
+stmtExecute_test({Safe, _Context, Conn}) ->
```

Comment:
Rename: `stmtExecute_test` -> `stmtExecute_simple`
```erlang
?assertEqual(Type, 'DPI_NATIVE_TYPE_DOUBLE')
```
change to
```erlang
?assertEqual('DPI_NATIVE_TYPE_DOUBLE', Type)
```
Ref: http://erlang.org/doc/apps/eunit/chapter.html#Assert_macros
```erlang
assertEqual(Expect, Expr)
```
**Fix all such patterns**